packages/clerk-js/src/utils/windowNavigate.ts

```diff
@@ -8,16 +11,13 @@ export const ALLOWED_PROTOCOLS = [
   'chrome-extension:',
 ];
 
+/**
+ * Helper utility to navigate via window.location.href. Also dispatches a clerk:beforeunload custom event.
+ *
+ * Note that this utility should **never** be called with a user-provided URL. We make no specific checks against the contents of the URL here and assume it is safe. Use `Clerk.navigate()` instead for user-provided URLs.
+ */
```

Comment:
Should we replace the usage for `windowNavigate` from SignIn.ts and SignUp.ts with `Clerk.navigate()` ?

Reply:
I don't think we need to, as the url is provided from our API and it will always be external.